src/onboarding2/Welcome/Suggestions.js

```diff
@@ -0,0 +1,103 @@
+import React, { useCallback, useMemo } from 'react'
+import { EthIdenticon, Box, GU } from '@aragon/ui'
+import FavoritesMenu from '../../components/FavoritesMenu/FavoritesMenu'
+import {
+  getKnownOrganization,
+  getRecommendedOrganizations,
+} from '../../known-organizations'
+import { network } from '../../environment'
+import { useFavoriteDaos } from '../../contexts/FavoriteDaosContext'
+
+const MAX_SUGGESTIONS = 6
+const RECOMMENDED_ORGS = getRecommendedOrganizations(network.type)
+
+function Suggestions() {
+  const {
+    favoriteDaos: favoriteOrgs,
+    isAddressFavorited,
+    removeFavoriteByAddress,
+    addFavorite,
+  } = useFavoriteDaos()
+
+  const updateFavorite = useCallback(
+    (address, favorite) => {
+      const org =
+        favoriteOrgs.find(org => org.address === address) ||
+        RECOMMENDED_ORGS.find(org => org.address === address)
+
+      // Can’t find the org
+      if (!org) {
+        return
+      }
+
+      if (favorite) {
+        addFavorite(org)
+      } else {
+        removeFavoriteByAddress(org.address)
+      }
+    },
+    [addFavorite, removeFavoriteByAddress, favoriteOrgs]
+  )
+
+  const suggestedOrgs = useMemo(() => {
+    const orgs = [...favoriteOrgs]
+
+    // Keep filling with recommended orgs until we reach the max
+    let i = 0
+    while (i < RECOMMENDED_ORGS.length && orgs.length < MAX_SUGGESTIONS) {
+      if (
+        orgs.findIndex(org => org.address === RECOMMENDED_ORGS[i].address) ===
+        -1
+      ) {
+        orgs.push(RECOMMENDED_ORGS[i])
+      }
+      i++
+    }
+    return orgs.sort((a, b) => (a.address > b.address ? 1 : -1))
+  }, [favoriteOrgs])
+
+  const openDao = useCallback(
+    address => {
+      const dao = suggestedOrgs.find(org => org.address === address)
+      if (dao) {
+        window.location.hash = `/${dao.name || dao.address}`
+      }
+    },
+    [suggestedOrgs]
+  )
+
+  return (
+    <Box heading="Suggestions" padding={false}>
+      <FavoritesMenu
+        items={suggestedOrgs.map(org => {
```

Comment:
I got the same org twice, is it valid or should we filter uniques?

![image](https://user-images.githubusercontent.com/3072458/64344488-df7a2180-cfee-11e9-8c4f-a16a1a716676.png)

Reply:
No this shouldn’t happen :thinking: Could you send me your `favorite-daos:rinkeby` value?

Reply:
Sure:
```
[{"name":"dev.aragonid.eth","address":"0x52d709f95d940d474d01c6f50d7686c3eff03831"},{"name":"a1.aragonid.eth","address":"0x837354c9dbeb4169157ef335762367e350e90eed"}]
```

Reply:
Of course, it was the f*** checksums 😅